VK/VK/ViewControllers/FeedViewController.swift

```diff
@@ -3,45 +3,135 @@
 
 import UIKit
 
+/// Identifier
+enum Identifier: String {
+    case headerCellIdentifier = "newsHeader"
+    case textCellIdentifier = "newsText"
+    case imageCellIdentifier = "newsImage"
+    case footerCellIdentifier = "newsFooter"
+}
+
 final class FeedViewController: UIViewController {
     // MARK: - IBOutlet
 
     @IBOutlet private var tableView: UITableView!
 
     // MARK: - Private properties
 
-    private let reuseIdentifier = "NewsCell"
+    private var newsList: [News] = [
+        News(
+            userImage: "1",
+            username: "Tony Soprano",
+            date: "3 сентября",
+            newsText: "Allo",
+            newsImage: "4",
+            type: .textAndImage
+        ),
+        News(
+            userImage: "2",
+            username: "Tony Montana",
+            date: "3 сентября",
+            newsText: "Allo",
+            newsImage: "2",
+            type: .image
+        ),
+        News(
+            userImage: "1",
+            username: "Vito Andolini (Corleone)",
+            date: "3 сентября",
+            newsText: "Born in Cosenza, Italy in 1891, Frank Costello grew up in East Harlem",
+            newsImage: "3",
+            type: .text
+        ),
+        News(
+            userImage: "1",
+            username: "Al Capone",
+            date: "3 сентября",
+            newsText: "Allo",
+            newsImage: "4",
+            type: .textAndImage
+        ),
+        News(
+            userImage: "2",
+            username: "Frank Costello",
+            date: "3 сентября",
+            newsText: "Under his leadership, Accordo expanded the profitability of the mob",
+            newsImage: "1",
+            type: .text
+        )
+    ]
 
     // MARK: - UIViewController
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        setupTableView()
+        tableView.dataSource = self
```

Comment:
```suggestion
        tableView.dataSource = self
        tableView.delegate = self
```
delegate - высоту для каждой ячейки отрегулируй